src/routes/user/_user.transactions.tsx

```diff
@@ -0,0 +1,179 @@
+import { useSuspenseQuery } from '@tanstack/react-query';
+import { createFileRoute, useSearch } from '@tanstack/react-router';
+
+import { CircleX, Search } from 'lucide-react';
+
+import React, { useEffect, useState } from 'react';
+
+import { toast } from 'sonner';
+// import { RecordTxDialogAdmin } from 'src/components/RecordTxDialogAdmin';
```

Comment:
commented import?

Reply:
Should be removed - I believe.
Generally - we should try not to leave such commented-out code and debug console.logs

Reply:
It has been removed